src/js/structs/Item.js

```diff
@@ -14,4 +14,8 @@ module.exports = class Item {
 
     return this._itemData[key];
   }
+
+  toJSON() {
```

Comment:
Let's move this to `Service`, as we're introducing it exclusively for the services.